### PR TITLE
Update SharedArrayBuffer article

### PR DESCRIPTION
**Pull request by agektmr**
This update adds information about upcoming changes around COEP: credentialless and COOP: same-origin-allow-popups.

### CODE REVIEW

**Comment by agektmr** [comment]
cc: @lutzvahl

**Comment by lutzvahl** [comment]
Thanks Eiji for the update. I've added two comments

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 1]
```diff
@@ -27,7 +27,12 @@ web, but things are settling down. Here's what you need to know:
 - `SharedArrayBuffer` is currently available in Desktop Chrome, but from Chrome
   91 it will be limited to cross-origin isolated pages. If you don't think you
   can make this change in time, you can [register for an origin trial](#origin-trial) to retain
-  the current behavior until Chrome 93.
+  the current behavior until at least Chrome 96.
+- If you intend to enable cross-origin isolation to continue using
+  `SharedArrayBuffer` evaluate the impact this will have on other cross-origin
+  elements on your website, e.g. ad placements. Check if `SharedArrayBuffer` is
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:33]:
```suggestion
  elements on your website, such as ad placements. Check if `SharedArrayBuffer` is
```

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 2]
```diff
@@ -50,8 +55,33 @@ can gather data on requests that failed as a result of
 `Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy`.
 
 If you don't think you can make these changes in time for Chrome 91, you can
-[register for an origin trial](#origin-trial) to retain current Desktop
-Chrome behavior until Chrome 93.
+[register for an origin trial](#origin-trial) to retain current Desktop Chrome
+behavior until at least Chrome 96.
+
+{% Aside %}
+**Update on April 2021**
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:62]:
```suggestion
**Update, April 2021**
```

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 3]
```diff
@@ -50,8 +55,33 @@ can gather data on requests that failed as a result of
...
+
+We've been exploring ways to deploy `Cross-Origin-Resource-Policy` at scale, as
+cross-origin isolation requires all subresources to explicitly opt-in. And we
+have come up with the idea of going in the opposite direction: a new [COEP
+"credentialless" mode](https://github.com/mikewest/credentiallessness/) that
+allows loading resources without the CORP header by stripping all their
+credentials. We are working out how it should work in detail, but we hope this
+will lighten your burden of making sure the subresources are sending the
+`Cross-Origin-Resource-Policy` header.
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:71]:
```suggestion
credentials. We are figuring out the details of how it should work, but we hope this
will lighten your burden of making sure the subresources are sending the
`Cross-Origin-Resource-Policy` header.
```

File: site/en/blog/enabling-shared-array-buffer/index.md [hunk 4]
```diff
@@ -50,8 +55,33 @@ can gather data on requests that failed as a result of
...
+Also, it's known that the `Cross-Origin-Opener-Policy: same-origin` header will
+break integrations that require cross-origin window interactions such as OAuth
+and payments. To mitigate this problem, we are [exploring relaxing the
+condition](https://github.com/whatwg/html/issues/6364) to enable cross-origin
+isolation to `Cross-Origin-Opener-Policy: same-origin-allow-popups`. This way
+the communication with the window opened by itself will be possible.
+
+If you want to enable cross-origin isolation to use `SharedArrayBuffer` but are
+blocked by these challenges, we recommend [registering for an origin
+trial](#origin-trial) and wait until the new modes are available. We are not
```
Comment (mihajlija) [inline site/en/blog/enabling-shared-array-buffer/index.md:82]:
```suggestion
trial](#origin-trial) and waiting until the new modes are available. We are not
```

**Review by mihajlija — APPROVED** [review]

**Pull request merged**